protocol/communities/adaptors.go

```diff
@@ -29,11 +36,13 @@ func (o *Community) ToSyncInstallationCommunityProtobuf(clock uint64, communityS
 	return &protobuf.SyncInstallationCommunity{
 		Clock:          clock,
 		Id:             o.ID(),
+		PrivateKey:     pkb,
```

Comment:
What's the goal of adding the PrivateKey?

Reply:
For open communities, once you lose access to the control node device, the community could be lost forever. We eliminated private key syncing primarily because we aimed to ensure that there is only one control node when a community is imported onto another device. Now, only one device (identified by its installation_id) can act as the control node at any given time, and this is guaranteed by the protocol. When a community is imported onto a different device, that device notifies the previous control node that it has now taken over control. This has the advantage of ensuring that the community isn't lost if the device is misplaced.

Reply:
Ok, I get it. I'm not a fan of putting private keys on waku, but it's encrypted and also once the OwnerToken is minted, it's then no longer used.